Sources/SnapshotTesting/Strategy/Any.swift

```diff
@@ -30,8 +30,6 @@ private func snap<T>(_ value: T, name: String? = nil, indent: Int = 0) -> String
     description = value.snapshotDescription
   case (let value as AnySnapshotStringConvertible, _):
     return "\(indentation)- \(name.map { "\($0): " } ?? "")\(value.snapshotDescription)\n"
-  case (let value as CustomDebugStringConvertible, _):
```

Comment:
I think `CustomDebugStringConvertible` assumes a lot, and usually embeds the descriptions of other types, bailing out of things like `AnySnapshotStringConvertible`, so less of a good idea.